### PR TITLE
fix: Update state colors

### PR DESCRIPTION
**Pull request by ursucarina**
# TL;DR
| status  | before |  after |
| ------------- | ------------- |------------- |
| succeeded  | image  | image  |
| running  | image  | image  |
| failed  | image  | image  |
| aborted  | image  | image  |
| paused  | image | image |

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by jsonporter — APPROVED** [review]

**Comment by bcrant** [comment]
@ursucarina Hey Carina, would you be open to reconsidering or revising this change? I'm colorblind and find it very difficult to tell succeeded and failed jobs apart.
Screenshot 2025-11-02 at 8 10 52 PM

failed is currently #FFDAD6 RGB(255, 218, 214)
succeeded is currently #D1EABF RGB(209, 234, 191).

I'm no expert on web accessibility, but by my eyes, if you increase the saturation of both by 10% they are much easier to distinguish.
failed #FFC3BD RGB(255, 195, 189)
succeeded #C3EAA8 RGB(195, 234, 168)

Whether or not those colors look good to people who aren't me, or fit with the style, I haven't the foggiest. Normally I'd open a PR, but since colors are not my strong suite, I thought I'd ask you! Thanks!

**Comment by JamesHoover** [comment]
> @ursucarina Hey Carina, would you be open to reconsidering or revising this change? I'm colorblind and find it very difficult to tell succeeded and failed jobs apart. Screenshot 2025-11-02 at 8 10 52 PM
> 
> failed is currently #FFDAD6 RGB(255, 218, 214) succeeded is currently #D1EABF RGB(209, 234, 191).
> 
> I'm no expert on web accessibility, but by my eyes, if you increase the saturation of both by 10% they are much easier to distinguish. failed #FFC3BD RGB(255, 195, 189) succeeded #C3EAA8 RGB(195, 234, 168)
> 
> Whether or not those colors look good to people who aren't me, or fit with the style, I haven't the foggiest. Normally I'd open a PR, but since colors are not my strong suite, I thought I'd ask you! Thanks!

Hey @bcrant thanks for bringing this to our attention, I will take a look and make sure we pick some colors that look more distinct.
